src/main/webapp/app/shared/metis/postings-create-edit-modal/post-create-edit-modal/post-create-edit-modal.component.ts

```diff
@@ -13,14 +13,18 @@ const TITLE_MAX_LENGTH = 200;
     templateUrl: './post-create-edit-modal.component.html',
 })
 export class PostCreateEditModalComponent extends PostingsCreateEditModalDirective<Post> {
+    tags: string[];
+
     constructor(protected metisService: MetisService, protected modalService: NgbModal, protected formBuilder: FormBuilder) {
         super(metisService, modalService, formBuilder);
+        this.tags = this.posting?.tags ?? [];
     }
 
     /**
-     * resets the post title and post content
+     * resets the post title, post content, and post tags
      */
     resetFormGroup(): void {
+        this.tags = this.posting.tags ? this.posting.tags : [];
```

Comment:
Might as well use the same style as above
```suggestion
        this.tags = this.posting?.tags ?? [];
```